pkg/registry/core/pod/storage/eviction.go

```diff
@@ -158,23 +158,33 @@ func (r *EvictionREST) Create(ctx context.Context, name string, obj runtime.Obje
 			return nil
 		}
 
-		pdb := &pdbs[0]
-		pdbName = pdb.Name
+		var pdb *policyv1beta1.PodDisruptionBudget
 		refresh := false
 		err = retry.RetryOnConflict(EvictionsRetry, func() error {
-			if refresh {
-				pdb, err = r.podDisruptionBudgetClient.PodDisruptionBudgets(pod.Namespace).Get(pdbName, metav1.GetOptions{})
-				if err != nil {
-					return err
+			for outer := range []int{0, 1} {
```

Comment:
Can we make it clearer here that we are doing an initial run with dry-run=true followed by a run with dry-run=false?

Reply:
Used bool variable for iteration.